src/openforms/registrations/contrib/objects_api/tests/test_templatetags.py

```diff
@@ -119,3 +119,29 @@ def test_render_json_summary_no_submission(self):
         )
 
         self.assertEqual(rendered, "{}")
+
+
+class AsGeoJsonTests(TestCase):
+    def test_render_as_geo_json(self):
+        rendered = render_from_string(
+            "{% as_geo_json variables.0.value %}",
+            context={"variables": [{"value": [1.0, 2.0]}]},
+            backend=openforms_backend,
+            disable_autoescape=True,
+        )
+
+        expected = '{"type": "Point", "coordinates": [1.0, 2.0]}'
+
+        self.assertEqual(rendered, expected)
+
+    def test_render_as_geo_json_no_value(self):
+        rendered = render_from_string(
+            "{% as_geo_json variables.0.value %}",
+            context={"variables": [{"value": ""}]},
```

Comment:
I've checked on our test environment and no value currently indeed presents itself as an empty string.

Which is weird, `null` would make more sense but that's the scope for a different ticket... :thinking: